src/modules/displaymanager/main.py

```diff
@@ -327,27 +406,55 @@ def run():
     if "lightdm" in displaymanagers:
         if have_dm("lightdm", root_mount_point):
             if enable_basic_setup:
-                libcalamares.utils.target_env_call(['mkdir', '-p', '/run/lightdm'])
-
-                if libcalamares.utils.target_env_call(['getent', 'group', 'lightdm']) != 0:
-                    libcalamares.utils.target_env_call(['groupadd', '-g', '620', 'lightdm'])
-
-                if libcalamares.utils.target_env_call(['getent', 'passwd', 'lightdm']) != 0:
-                    libcalamares.utils.target_env_call(['useradd', '-c', '"LightDM Display Manager"',
-                                                    '-u', '620', '-g', 'lightdm', '-d', '/var/run/lightdm',
-                                                    '-s', '/usr/bin/nologin', 'lightdm'])
-
-                libcalamares.utils.target_env_call(['passwd', '-l', 'lightdm'])
-                libcalamares.utils.target_env_call(['chown', '-R', 'lightdm:lightdm', '/run/lightdm'])
-                libcalamares.utils.target_env_call(['chmod', '+r' '/etc/lightdm/lightdm.conf'])
+                libcalamares.utils.target_env_call(
+                    'mkdir', '-p', '/run/lightdm'
```

Comment:
You have lost the [] list brackets here and in the following bits .. I don't know if target_env_call wants to be called that way.